data/tests/fwupdtool-efiboot.sh

```diff
@@ -49,11 +49,21 @@ echo "Setting BootOrder"
 run efiboot-order 0001
 rc=$?; if [ $rc != 0 ]; then error $rc; fi
 
+# ---
+echo "Getting BootOrder"
+run efiboot-order
+rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
```

Comment:
this makes me realize there should probably also be a check for reading boot order before it's set (which I guess should error)